index.html

```diff
@@ -50,7 +50,7 @@
         }
         function _setDollarValues() {
             //USD
-            document.getElementById('current-price-dollar').innerText = _formatNumber($.priceInfo.price_usd) + ' $';
+            document.getElementById('current-price-dollar').innerText = '$' + _formatNumber($.priceInfo.price_usd).toFixed(2);
```

Comment:
Dollar sign format is correct, but you should take a look at the function '_formatNumber'. It should fomat the number to 2 decimal places.

Reply:
It looks like the current code formats it to a maximum of 2 decimal places, but if for example the price was $2.10 it was displaying as $2.1 - I didn't put the toFixed(2) in the _formatNumber function because I wasn't sure if it was used for anything other than currency. If _formatNumber is just for currency values then the toFixed(2) could be added onto the return value from there instead.